config/clusters/cloudbank/csum.values.yaml

```diff
@@ -0,0 +1,78 @@
+jupyterhub:
+  ingress:
+    hosts: [csum.cloudbank.2i2c.cloud]
+    tls:
+      - hosts: [csum.cloudbank.2i2c.cloud]
+        secretName: https-auto-tls
+  singleuser:
+    memory:
+      # Increased to help deal with possible kernel restarts
+      # https://2i2c.freshdesk.com/a/tickets/567
+      guarantee: 384M
+      limit: 1.5G
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "google"
+    homepage:
+      templateVars:
+        org:
+          name: CSU Maritime
+          logo_url: https://www.csum.edu/_resources/images/cal-maritime-logo.svg
+          url: https://www.csum.edu/
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: CloudBank
+          url: http://cloudbank.org/
+        funded_by:
+          name: CloudBank
+          url: http://cloudbank.org/
+  hub:
+    config:
+      JupyterHub:
+        authenticator_class: cilogon
+      CILogonOAuthenticator:
+        oauth_callback_url: "https://csum.cloudbank.2i2c.cloud/hub/oauth_callback"
+        allowed_idps:
+          https://cma-shibboleth.csum.edu/idp/shibboleth:
+            username_derivation:
+              username_claim: "email"
+            # allow_all is a partial authorization, username_pattern is enforced also
+            allow_all: true
+          http://google.com/accounts/o8/id:
+            username_derivation:
+              username_claim: "email"
+            # allow_all is a partial authorization, username_pattern is enforced also
+            allow_all: true
+      Authenticator:
+        admin_users:
+          - ericvd@berkeley.edu
+          - sean.smorris@berkeley.edu
+          - jteoh@csum.edu
```

Comment:
This configuration didn't include `username_pattern`, and with `allow_all` to true for the Google IdP, all users google users are allowed access @sean-morris.

I'll propose a fix for this in a PR

Reply:
ahhhh.. OK Thank you! so it should be allowed_domains: 2i2c and berkeley

Reply:
Yes that is an improvement, but that is also not optimal as many berkeley users not intended to have access are then still allowed.

I've proposed what i think is a suitable solution in #3232 where we dont have allowed_domains or allow_all, instead its now enough with listing the usernames to allow access via admin_users.

If you find time time, please review that pull request!